pkg/vpcmodel/output.go

```diff
@@ -140,10 +202,15 @@ func WriteToFile(content, fileName string) (string, error) {
 
 // AggregateVPCsOutput returns the output string for a list of SingleAnalysisOutput objects
 // and writes the output to outFile
-func AggregateVPCsOutput(outputList []*SingleAnalysisOutput, f OutFormat, uc OutputUseCase, outFile string) (string, error) {
+func (of *serialOutputFormatter) AggregateVPCsOutput(outputList []*SingleAnalysisOutput, uc OutputUseCase, outFile string) (string, error) {
 	var res string
 	var err error
-	switch f {
+
+	sort.Slice(outputList, func(i, j int) bool {
+		return outputList[i].VPC1Name < outputList[j].VPC1Name
+	})
+
```

Comment:
This sorting was done in main before, right?

Reply:
was done in testing, so the output will be as expected